client/src/components/App/Content/LandingPage/InvestigationTable/TableFilter/TableFilter.tsx

```diff
@@ -1,82 +1,134 @@
 import React from 'react'
 import { Close } from '@material-ui/icons';
 import { Autocomplete } from '@material-ui/lab';
-import { Card, Checkbox, IconButton, TextField, Typography } from '@material-ui/core';
+import { Card, Checkbox, Collapse, FormControl, Grid, IconButton, TextField, Typography } from '@material-ui/core';
 
+import { TimeRange } from 'models/TimeRange';
+import timeRanges from 'models/enums/timeRanges';
+import DateRangePick from 'commons/DatePick/DateRangePick';
+import SelectDropdown from 'commons/Select/SelectDropdown';
 import InvestigationMainStatus from 'models/InvestigationMainStatus';
 
 import useStyles from './TableFilterStyles';
+import useTableFilter from './useTableFilter';
 import { StatusFilter as StatusFilterType } from '../InvestigationTableInterfaces';
 
-interface Props {
-    statuses: InvestigationMainStatus[];
-    filteredStatuses: StatusFilterType;
-    unassignedUserFilter: boolean;
-    inactiveUserFilter: boolean;
-    changeUnassginedUserFilter: (isFilterOn: boolean) => void;
-    changeInactiveUserFilter: (isFilterOn: boolean) => void;
-    onFilterChange: (event: React.ChangeEvent<{}>, selectedStatuses: InvestigationMainStatus[]) => void;
-    onClose: () => void;
-}
+const customTimeRangeId = -1;
+const timeRangeMinDate = new Date(2020, 9, 1);
```

Comment:
doesn't those fields exist in another page? maybe export them to Utils or something

Reply:
I exported them because I do use them in 2 places